app/(auth)/business/form.tsx

```diff
@@ -67,11 +66,106 @@ const formSchema = z.object({
   businessType: z.enum(businessTypes),
   businessName: z.string().min(3),
   country: z.enum(countries),
-  stripeDashboardAccess: z.enum(stripeDashboardAccess),
+  stripeDashboardType: z.enum(stripeDashboardTypes),
   paymentLosses: z.enum(paymentLosses),
-  feePayer: z.enum(feePayer),
+  feePayer: z.enum(feePayers),
 });
 
+function StripeFeeCollectionSelect({
+  field,
+  feePayerLabels,
+  validFeePayers,
+}: {
+  field: ControllerRenderProps<z.infer<typeof formSchema>, 'feePayer'>;
+  feePayerLabels: Record<string, string>;
+  validFeePayers: readonly FeePayer[];
+}) {
+  return (
+    <Select {...field} onValueChange={(value) => field.onChange(value)}>
+      <SelectTrigger>
+        <SelectValue>{feePayerLabels[field.value]}</SelectValue>
+      </SelectTrigger>
+      <SelectContent>
+        {feePayers.map((option) => (
+          <SelectItem
+            key={option}
+            value={option}
+            disabled={!validFeePayers.includes(option)}
+          >
+            {feePayerLabels[option]}
+          </SelectItem>
+        ))}
+      </SelectContent>
+    </Select>
+  );
+}
+
+function NegativeBalanceLiabilitySelect({
+  field,
+  validPaymentLosses,
+}: {
+  field: ControllerRenderProps<z.infer<typeof formSchema>, 'paymentLosses'>;
+  validPaymentLosses: readonly string[];
+}) {
+  return (
+    <Select {...field} onValueChange={(value) => field.onChange(value)}>
+      <SelectTrigger>
+        <SelectValue>{paymentLossesLabels[field.value]}</SelectValue>
+      </SelectTrigger>
+      <SelectContent>
+        {paymentLosses.map((option) => (
+          <SelectItem
+            key={option}
+            value={option}
+            disabled={!validPaymentLosses.includes(option)}
+          >
+            {paymentLossesLabels[option]}
+          </SelectItem>
+        ))}
+      </SelectContent>
+    </Select>
+  );
+}
+
+function StripeDashboardTypeSelect({
+  field,
+  stripeDashboardTypes,
+  stripeDashboardTypeLabels,
+}: {
+  field: ControllerRenderProps<
+    z.infer<typeof formSchema>,
+    'stripeDashboardType'
+  >;
+  stripeDashboardTypes: readonly StripeDashboardType[];
+  stripeDashboardTypeLabels: Record<StripeDashboardType, string>;
+}) {
+  return (
+    <Select
+      {...field}
+      onValueChange={(value) => {
+        // if (value === 'express') {
```

Comment:
why is this commented out?

Reply:
Oh it's meant to be removed, I just missed it